modules/drivers/snowflake/src/metabase/driver/snowflake.clj

```diff
@@ -406,28 +405,62 @@
   (sql-jdbc/query driver database {:select [:*]
                                    :from   [[(qp.store/with-metadata-provider (u/the-id database)
                                                (sql.qp/->honeysql driver table))]]}))
+(defn- show-command-sql
+  ;; IMPORTANT: prefer using this command over using the JDBC `.getTables` method because the jdbc is buggy
+  ;; it works sometimes but other times randomly.
+  ;; we have tried to use it but people reported that some schemas disappear on their instances.
+  ;; See https://discourse.metabase.com/t/snowflake-broken-in-0-48-1/63849
+  [object-type & {:keys [like database schema]}]
```

Comment:
Making this an `mu/defn` and giving `object-type` and `:enum` schema would make it a lot clearer for people using this what kinds of things it supports. A docstring with an example usage would be helpful too